app/Services/Monitor/DelegateTracker.php

```diff
@@ -430,5 +431,17 @@ public static function execute(Collection $delegates, int $startHeight): array
                 ];
             })
             ->toArray();
+
+        // TODO: static order will be based on start height only, all other times we will need to only update the forging time and nothing else
+        $originalOrder = ForgingInfoCalculator::calculate(Block::where('height', $startHeight)->firstOrFail()->timestamp, $startHeight);
+
+        // Note: static order will be found by shifting the index based on the forging data from above
+        $delCount         = count($activeDelegates); // TODO: fetch this from somewhere as it almost never changes
```

Comment:
```suggestion
        $delegateCount         = Network::delegateCount();
```

This would probably suffice? Usually genesis would cover if there are no other delegates to fill spots.

Reply:
:vinigotit: